src/internal/hooks/useMessageReceivers.ts

```diff
@@ -62,7 +62,10 @@ export const useCommonMessageReceivers = (
   );
 
   useReceiveMessage("getThemeData", TARGET_ORIGINS, (send, payload) => {
-    const themeData = jsonFromEscapedJsonString(payload as unknown as string);
+    const payloadString = payload as unknown as string;
+    const themeData = payloadString
```

Comment:
Does no content return nothing, opposed to an empty object?

Reply:
Yes it returns nothing

This was needed in order to handle NoContent